features/tracker-allowlist.json

```diff
@@ -1879,7 +1880,8 @@
                         ],
                         "reason": [
                             "https://github.com/duckduckgo/privacy-configuration/issues/1173",
-                            "triblive.com - https://github.com/duckduckgo/privacy-configuration/issues/1730"
+                            "triblive.com - https://github.com/duckduckgo/privacy-configuration/issues/1730",
+                            "pointstreaksites.com - https://github.com/duckduckgo/privacy-configuration/issues/2197"
```

Comment:
This reason is pointing to this PR, which doesn’t show a reason for this site. I think the reason should be made clear in the description of this PR (unless it’s already in a different PR, in which case the GitHub URL should be updated).

Reply:
@icodebyamanda Actually I see the reason now. Not sure whether you’ve updated or I just didn’t read it properly (likely!), but all good IMO.